extension/audit.go

```diff
@@ -87,16 +87,22 @@ func (a AuditLogger) InterceptField(ctx context.Context, next graphql.Resolver)
 
 	if fc.IsResolver {
 		if callBy == "Query" || callBy == "Mutation" || callBy == "Subscription" {
-			if _, found := m.Resolvers[key]; !found {
-				m.Resolvers[key] = true
+			v, found := m.Resolvers[key]
+			if found {
+				m.Resolvers[key] = v + 1
+			} else {
+				m.Resolvers[key] = 1
```

Comment:
I prefer this one.
```
m.Resolvers[key]++
```

Reply:
Thanks! fixed in https://github.com/jlwt90/gqlgen-usage-analysis/pull/2/commits/ed3d7f0a4531231f47b86ae536b0b6c3e3bafdb2